chain/app/db_app.go

```diff
@@ -71,6 +71,24 @@ func (app *dbBandApp) InitChain(req abci.RequestInitChain) abci.ResponseInitChai
 				err := app.dbBand.AddValidator(
 					createMsg.ValidatorAddress,
 					createMsg.PubKey,
+					createMsg.Description.Moniker,
+					createMsg.Description.Identity,
+					createMsg.Description.Website,
+					createMsg.Description.Details,
+					createMsg.Commission.Rate.String(),
+					createMsg.Commission.MaxRate.String(),
+					createMsg.Commission.MaxChangeRate.String(),
+					sdk.NewCoin("uband", createMsg.MinSelfDelegation).String(),
```

Comment:
Let's not hard-code `uband`. We can just store it add is. The frontend will decide if it wants to add the unit.